onlyswaps-verifier/local-verifier-config.json

```diff
@@ -0,0 +1,33 @@
+{
+  "agent": {
+    "healthcheck_listen_addr": "0.0.0.0",
+    "healthcheck_port": 9999
+  },
+
+  "networks": [{
+    "chain_id": 31337,
+    "rpc_url": "ws://localhost:31337",
+    "router_address": "0x1293f79c4fa7fa83610fa5ef8064ef64929ee2fd",
+    "private_key": "0x868c3482353618000889b0e733022108e174bb821e1fdb43bb56dc8115e218d2"
```

Comment:
I'm unclear on why the _verifier_  needs a private key, tbh.

Reply:
hmm that's an interesting idea - they could even aggregate multiple signatures to batch redeem their fees 🤔 for v0, I suggest we ship this version (and Randamu sponsors all the redemption gas fees for now). We've talked about a few incentivisation designs, but the solver redemption one makes the most sense imo